packages/BarChart/src/BarChart.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import cx from 'classnames';
+
+import './bar-chart.scss';
+
+export interface DataSet {
+  /** Add label text above the bar chart. */
+  label: string;
+  /** The value displayed beside the bar chart. */
+  value: number;
+  /** Add the color to the bar chart. */
+  color?: string;
+}
+
+export interface BarChartProps extends React.ComponentProps<'li'> {
+  /** The data object that will be used for the bar chart. */
+  dataSet: DataSet[];
+  /** Add a data-test tag for automated tests. */
+  dataTest?: string;
+  /** Add a specific class to the component. */
+  className?: string;
+}
+
+const BarChart: React.FunctionComponent<BarChartProps> = (
+  props: BarChartProps
+) => {
+  const { dataSet, className, dataTest, ...rest } = props;
+
+  if (!dataSet) {
+    return null;
+  }
+
+  const setWidth = (value: number) => {
```

Comment:
Missing method return type
```suggestion
  const setWidth = (value: number): string | undefined => {
```